app/bundles/InstallBundle/DependencyInjection/Compiler/InstallCommandPass.php

```diff
@@ -0,0 +1,54 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Mautic\InstallBundle\DependencyInjection\Compiler;
+
+use Mautic\InstallBundle\Command\InstallCommand;
+use Symfony\Component\Console\Input\ArgvInput;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\DependencyInjection\Compiler\CompilerPassInterface;
+use Symfony\Component\DependencyInjection\ContainerBuilder;
+
+class InstallCommandPass implements CompilerPassInterface
+{
+    public function process(ContainerBuilder $container): void
+    {
+        $args = $_SERVER['argv'] ?? [];
+
+        if (!in_array(InstallCommand::COMMAND, $args, true)) {
+            return;
+        }
+
+        $definition = (new InstallCommand())->getDefinition();
+        $definition->addOption(
+            new InputOption('--verbose', '-v', InputOption::VALUE_NONE, 'Increase verbosity of messages.')
+        );
+        $definition->addOption(
+            new InputOption(
+                '--env',
+                '-e',
+                InputOption::VALUE_REQUIRED,
+                'The Environment name.',
+                $container->getParameter('kernel.environment')
+            )
+        );
```

Comment:
@mabumusa1 what was the reason the add the options here and not in the `InstallCommand` itself?

Reply:
@escopecz I think we needed this to be injected before the cache warmup, otherwise, it will not be collected if inserted in the `InstallCommand`. I am not 100% sure. but I think this is the reason

Reply:
I totally agree, this PR was not fully covered. Once I am done with the email PR I will work on increasing the coverage for this command